src/main/resources/lang/quilt_loader_es_AR.properties

```diff
@@ -0,0 +1,132 @@
+button.open_mods_folder=Abrir carpeta de mods
+button.continue=Continuar
+button.ignore=Ignorar
+button.reload=Recargar
+button.continue_to=Continuar a %s
+button.exit=Salir
+# Warnings
+warn.sub_folders_disabled=Cargar desde sub-carpetas está deshabilitado, asi que esta carpeta fue ignorada
+warn.sub_folder_ignored=Esta carpeta tiene un archivo "quilt_loader_ignored", así que esta carpeta fue ignorada
+warn.unhandled_mod=¡Ningún plugin se pudo cargar a partir de esto!
+warn.invalid_version_specifier=¡Versión de juego inválida!
+# Errors
+crash.during_setup.minecraft=Minecraft falló a iniciarse!
+error.failed_to_save_crash_report=¡No se pudo guardar el informe de choque!
+error.failed_to_save_crash_report.desc=¡Esto es problabemente malo! Por favor copia el informe de choque y guárdalo manualmente a caso que lo necesites\nO fíjate si está registrado en el archivo de registro
+error.failed_to_save_crash_report.info=Quilt-loader no sabe donde los registros son guardados :(
+error.unhandled=¡Error Interno!
+error.unhandled.desc=Algo muy malo pasó mientras se cargaban mods\nPor favor informe este error al proyecto quilt-loader para que esto se solucione.
+error.unhandled_launch.desc=SAlgo muy malo pasó cuando se ejecutaba el juego\nPor favor informe error a los foros quilt asi podamos ayudar a arreglarlo.
+error.too_many_errors=¡Demasiados errores para mostrar!
+error.too_many_errors.desc=Por favor arregla los errores mostrados hacia abajo, y después intenta para ver errores adicionales.\n(%s errores tradicionales)
+error.quilt_overrides.io_parse.title= ¡Se ha fallado la lectura del archivo de omisión!
+gui.text.ioexception_files_hidden=No se pudo verificar si un archivo estaba oculto: %s
+gui.text.ioexception_files_hidden.desc.0=%s
+gui.error.zipexception=Se ha falladod a leer como un zip: %s
+gui.error.zipexception.title=Archivo zip roto: %s
+gui.error.zipexception.desc.0=%s
+gui.error.zipexception.desc.1=¿Intentar descargar el archivo de nuevo?
+gui.error.ioexception=Se falló la lectura: %s
+gui.error.ioexception.title=No se ha podido leer: %s
+gui.error.ioexception.desc.0=%s
+gui.text.invalid_metadata=Metadata inválida: %s
+gui.text.invalid_metadata.title=No se ha podido leer%s: %s
+gui.text.invalid_metadata.desc.0=Desde %s
+error.unhandled_solver=Error de solucionador complejo; fijarse el informe de choque para más informacion
+error.unhandled_solver.desc= Por favor intenta actualizar quilt-loader para ver si una actualización puede describir esto\nO informe esto al projecto quilt-loader para que sea arreglado.
+error.unhandled_solver.desc.rule_n= \n \nReglas %s (%s):
+fabric.jar_in_jar.missing=Included file not found! ¡Archivos incluidos no encontrados!
+error.duplicate_mandatory=Mod duplicado: %s
+error.duplicate_mandatory.mod=- %s
+error.duplicate_mandatory.desc=Remover todos excepto uno.
+error.dep.join.title=%s, %s
+
+error.dep.direct.any.title=%s require cualquier versíon de %s
```

Comment:
All the titles changes the "require" sentence, sometimes says "require" sometimes "requiere" sometime "requieres" or "requires"